plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/MethodSignatureEmbedding.kt

```diff
@@ -21,7 +22,10 @@ class MethodSignatureEmbedding(
     val params: List<VariableEmbedding>,
     val returnType: TypeEmbedding,
 ) {
-    val name = symbol.callableId.embedName()
+    val name = when (symbol) {
+        is FirPropertyAccessorSymbol -> symbol.propertySymbol.callableId.embedName()
```

Comment:
Is this correct?  Doesn't this encode both the getter and the setter using the same name?
I would be more inclined to put this logic into `embedName`.